lib/modules/manager/nix/extract.spec.ts

```diff
@@ -639,6 +648,159 @@ describe('modules/manager/nix/extract', () => {
           datasource: 'git-refs',
           depName: 'subgroup-project',
           packageName: 'https://gitlab.com/group/sub-group/subgroup-project',
+          rangeStrategy: 'update-lockfile',
+        },
+      ],
+    });
+  });
+
+  const flake13Lock = `{
```

Comment:
move inside the test and use `codeBlock` string template helper

Reply:
#35391 makes rebasing this kinda hard ...